app/liquid_tags/raw_tag.rb

```diff
@@ -0,0 +1,7 @@
+module Liquid
+  class Raw < Block
+    FullTokenPossiblyInvalid = /\A(.*)#{TagStart}\s*(\w+)\s*#{TagEnd}\z/om
```

Comment:
Renaming the constant like this prints the following warning:
```
dev.to/app/liquid_tags/raw_tag.rb:3: warning: already initialized constant Liquid::Raw::FullTokenPossiblyInvalid
.rbenv/versions/2.5.3/lib/ruby/gems/2.5.0/gems/liquid-4.0.1/lib/liquid/tags/raw.rb:4: warning: previous definition of FullTokenPossiblyInvalid was here
```

Not sure how to get around it, but I think this warning will only at load time.

Reply:
I can't find it right now but there's a sensible way to reassign class's constant so that warning appear everywhere.